awx/ui_next/src/screens/Inventory/Inventories.jsx

```diff
@@ -57,6 +57,13 @@ class Inventories extends Component {
       ),
       [`/inventories/inventory/${inventory.id}/sources`]: i18n._(t`Sources`),
       [`/inventories/inventory/${inventory.id}/groups`]: i18n._(t`Groups`),
+      [`/inventories/inventory/${inventory.id}/groups/add`]: i18n._(
+        t`Create New Group`
+      ),
```

Comment:
Adding the path and name below will include the group name into the breadcrumb. 
[`/inventories/inventory/${inventory.id}/groups/${group && group.id}`]: `${group && group.name}`,